src/io/show/MetabolicModel.jl

```diff
@@ -3,7 +3,7 @@ Pretty printing of everything metabolic-modelish.
 """
 function Base.show(io::IO, ::MIME"text/plain", m::MetabolicModel)
     _pretty_print_keyvals(io, "", "Metabolic model of type $(typeof(m))")
-    if prod(size(stoichiometry(m))) < 5_000_000
+    if prod(size(stoichiometry(m))) < 20_000_000
```

Comment:
shall we not have this in the constants?

Reply:
done!